code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -156,10 +156,10 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 
 /datum/gas/freon
 	id = "freon"
-	specific_heat = 600
+	specific_heat = 2069
```

Comment:
Somewhat similar to what ghil was saying, why pick a number this large? it outclasses our current max, nobilium, by 69 `nice`. Why this high? plasma, the second best, is at 200, this just seems extreme.

Reply:
It's a very cold gas

Reply:
Being cold is not a reason to have that amount of heat capacity